js/script.js

```diff
@@ -50,7 +45,11 @@
                 position: cords,
                 map: map
             });
+            var id = items[i].id;
+            marker.addListener('click', function(){
+                flkty.selectCell(id);
+                console.log(flkty)
```

Comment:
zamiast `console.log(flkty)`, ktore pewnie zbyt wiele nie mowi, wyswietl sobie `console.log(id)`, zobaczysz, ze w momencie klikania na markery zmienna `id` zadeklarowana poza callbackiem event listenera wyswietla zawsze to, co po zakonczeniu dzialania petli, czyli `id` przyjmuje ostatnia wartosc `items[i].id`